releasenotes/notes/add-xxminusyy-gate-63e6530c23500de9.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Adds a gate for the XX-YY interaction, accessible via
+    `qiskit.circuit.library.XXMinusYYGate`. This gate can be used
```

Comment:
Isn't the `:class:` qualifier required to properly link it as class in Sphinx?

Reply:
Done.